docs/brisket.responseobject.md

```diff
@@ -59,6 +59,19 @@ Here is a listing of all the tools provided by the Brisket response object.
 ### response.status(statusCode)
 Use this method to set the statusCode for the current response.
 
+### response.set(field [, value])
+Use this method to set the response's HTTP header `field` to `value`. You can set multiple values by passing an object.
```

Comment:
in this case, i don't think knowing the details about what happens on the client adds enough value since you already know the effect. since response headers don't have a meaning as you navigate a single page app (currently), it's not so important that the docs communicate that nothing happens.